server/scripts/exportstandalonedeck.js

```diff
@@ -0,0 +1,36 @@
+/*eslint no-console:0 */
+
+const monk = require('monk');
+
+const CardService = require('../services/CardService');
+const DeckService = require('../services/DeckService');
+
+class ExportStandaloneDecks {
+    constructor() {
+        this.db = monk('mongodb://127.0.0.1:27017/throneteki');
```

Comment:
does this have to be changed depending on the dev´s setup or should we use the property dbPath from the local/default.json5 config file here?

Reply:
Going to leave it for the moment, as the same thing exists in `ImportStandaloneDecks` (which is run / is meant for even production)